vagrant/lib/forklift/box_distributor.rb

```diff
@@ -286,12 +289,16 @@ def configure_rackspace(machine, box)
 
     def configure_openstack_provider(machine, box)
       machine.vm.provider :openstack do |p, override|
-        override.vm.box        = nil
-        override.ssh.username  = 'root'
+        override.vm.box = nil
+        if box.fetch('sync_disabled', nil)
+          override.vm.synced_folder '.', '/home/vagrant/sync', disabled: true
```

Comment:
The default sync target is `/vagrant`. See the libvirt example:

https://github.com/theforeman/forklift/blob/01c5da2452ef781c3dea2b73e113edf4e48daab5/vagrant/lib/forklift/box_distributor.rb#L214